examples/app/export_csv/download.js

```diff
@@ -1,14 +1,23 @@
-var data = source.data;
-var filetext = 'name,income,years_experience\n';
-for (var i = 0; i < data['name'].length; i++) {
-    var currRow = [data['name'][i].toString(),
-                   data['salary'][i].toString(),
-                   data['years_experience'][i].toString().concat('\n')];
+function table_to_csv(data_table) {
+    var columns = Object.keys(data_table)
+    var nrows = data_table[columns[0]].length
+    var lines = [columns.join(',')];
```

Comment:
Should consistently not use semicolons throughout. Also should use `const` and `let` instead of `var` anywhere

Reply:
Ok, I think this is fixed now.

There is one line `filetext = table_to_csv(data)` that doesn't have `const`, `let`, or `var` now (and it doesn't seem to happen inside the function either). It still works, but if it's bad style can you let me know what I should replace this with? Many thanks!